### PR TITLE
Completed GitHub community standards

### PR DESCRIPTION
**Pull request by Veldergard**
My first pull request

Issue #420 
- Code of conduct
- Security policy
- Issue template
- Pull request template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+name: 🐛 Bug
+about: Create a report to help us improve Blaze
+title: ''
+labels: ''
+assignees: ''
+---
+<!--
+Before reporting a bug, please check for existing or closed issues
+first and read the instructions for filing a bug report:
+https://github.com/meteor/meteor/blob/devel/CONTRIBUTING.md#reporting-a-bug-in-meteor
```
Comment (jankapunkt) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md:11]: We should change this to:

> Before reporting a bug, please check for existing or closed issues
first.

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### This bug report should include:
+- [ ] A short, but descriptive title. The title doesn't need "Blaze" in it.
+- [ ] The version of Blaze showing the problem.
+- [ ] The last version of Blaze where the problem did _not_ occur, if applicable.
+- [ ] The operating system you're running Blaze on.
+- [ ] The expected behavior.
+- [ ] The actual behavior.
+- [ ] A **simple** reproduction! (Must include the Github repository and steps to reproduce the issue on it.)
```
Comment (jankapunkt) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md:20]: We should change `(Must include the Github repository and steps to reproduce the issue on it.)` to

> (Must either include a minimal code example or a link to a  Github repository with steps to reproduce the issue on it.)

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+If you don't include a reproduction the issue is probably going to be closed.
+
+### Independent packages
```
Comment (jankapunkt) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md:24]: This section can be removed entirely

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Support Status
```
Comment (jankapunkt) [inline SECURITY.md:5]: Please change this to

```
| Version | Support Status
| ------- | --------------
| 3.x.y   | ✅  all security issues
| >=2.5   | ✅  all security issues
| <2.5   | 🚧  only critical security issues
| <= 2  | ❌  no longer supported
```

**Review by jankapunkt — CHANGES_REQUESTED** [review]
Please change the commented sections, from my end the rest looks good. I think this still needs at least approval from a core member @Grubba27 @denihs especially on the Security Policy and if reporting to security@meteor.com is the right action here.

**Comment by jankapunkt** [comment]
@Veldergard in order to update this PR with your changes you simply implement your changes, commit them and push them to this branch, then this page will automatically update.

**Comment by Veldergard** [comment]
@jankapunkt i commited changes.
@Grubba27 @denihs check for other issues pls

**Review by StorytellerCZ — APPROVED** [review]

**Comment by Grubba27** [comment]
Talked with @fredmaiaarantes and is okay to use [security@meteor.com](mailto:security@meteor.com)

**Comment by jankapunkt** [comment]
Congratulations @Veldergard to your first merged pull request :tada:

**Pull request merged**